src/tape/ensemble_readers.py

```diff
@@ -29,6 +37,19 @@ def read_ensemble(dirpath, additional_frames=True, column_mapper=None, dask_clie
         Supplies a ColumnMapper to the Ensemble, if None (default) searches
         for a column_mapper.npy file in the directory, which should be
         created when the ensemble is saved.
+    additional_cols: 'bool', optional
+        Boolean to indicate whether to carry in columns beyond the
+        critical columns, true will, while false will only load the columns
+        containing the critical quantities (id,time,flux,err,band)
+    partition_size: `int`, optional
+            If specified, attempts to repartition the ensemble to partitions
+            of size `partition_size`.
+    sorted: bool, optional
+        If the index column is already sorted in increasing order.
+        Defaults to False
+    sort: `bool`, optional
+        If True, sorts the DataFrame by the id column. Otherwise set the
+        index on the individual existing partitions. Defaults to False.
```

Comment:
Do we want `sort` and `sorted` to be explicit parameters in the function definition? Or did we mean to remove these from the docstring?

Reply:
Good catch, forgot to remove these from ensemble_readers.py as well!